src/i965_drv_video.c

```diff
@@ -974,12 +974,18 @@ i965_GetConfigAttributes(VADriverContextP ctx,
                     attrib_list[i].value |= VA_RC_CBR;
 
                 if (profile == VAProfileVP8Version0_3 ||
-                    profile == VAProfileVP9Profile0 ||
-                    profile == VAProfileH264ConstrainedBaseline ||
+                    profile == VAProfileVP9Profile0)
+                    attrib_list[i].value |= VA_RC_VBR;
```

Comment:
Cherryview can support vp8 encoder, but without new kernel, right? So it should not support VBR.